### DISCUSSION TITLE
"Sorry, we have no imagery here” shown

### DISCUSSION THREAD

**Discussion by fabriziogiudici** [dlsc-software-consulting-gmbh/GMapsFX#208] [General]
Hello.

I'm trying out GMapsFX integrated in an app of mine, but tiles show the error message "Sorry, we have no imagery here” everywhere, at every zoom level. I suppose it's a thing of configuration, perhaps I need to specify an API key? But how to do that in the GMapsFX API?
Thanks.

**Comment by danielwilbanks** [comment]
See this issue: 

https://github.com/dlsc-software-consulting-gmbh/GMapsFX/issues/207

To my knowledge nothing further has been figured out.

**Comment by fabriziogiudici** [comment]
Thanks.
